Reference/Scheduling/index-v8.md

```diff
@@ -99,6 +99,10 @@ namespace Umbraco.Web.UI
 
 ```
 
+:::tip
+Trying to inject services or helpers that rely on `UmbracoContext` such as `UmbracoHelper` or `MembershipHelper` will trigger a boot failed error on startup. See the [Services and Helpers](../../Implementation/Services/index-v8.md#accessing-published-content-outside-of-a-http-request) article on how to get around this by using the `UmbracoContextFactory`.
```

Comment:
Suggestion: Rename the link to the section title rather than the article name. Also rephrase the sentence to something similar for clarity:

See the [Accessing Published Content outside of a Http Request](../../Implementation/Services/index-v8.md#accessing-published-content-outside-of-a-http-request) article to query the Umbraco Published Content using `UmbracoContextFactory`.

Reply:
I've processed your feedback in the latest commit.